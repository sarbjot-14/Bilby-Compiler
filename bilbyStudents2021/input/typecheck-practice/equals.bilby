main {
    imm resultthree := 86 == 86.0;
    print resultthree $n;
}
