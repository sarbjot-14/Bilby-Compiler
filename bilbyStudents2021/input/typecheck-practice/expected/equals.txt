Bad operand type for StoreI instruction at location 40.
